src/frontend/src/pages/DRCPage/DRCPage.tsx

```diff
@@ -0,0 +1,141 @@
+
+const DRCPage = () => {
+  const theme = useTheme();
+
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const [displayMonth, setDisplayMonth] = useState<Date>(new Date());
+
+  const EventDict = new Map<Number, DesignReview[]>();
+  // Test data:
+  EventDict.set(new Date().getDate(), [
+    {
+      designReviewId: 'Meeting',
+      dateScheduled: new Date(),
+      meetingTimes: [16],
+      dateCreated: new Date(),
+      userCreated: batman,
+      status: DesignReviewStatus.UNCONFIRMED,
+      teamType: { teamTypeId: 'Mechanical', name: 'Mechanical' },
+      requiredMembers: [],
+      optionalMembers: [],
+      confirmedMembers: [],
+      deniedMembers: [],
+      isOnline: false,
+      isInPerson: false,
+      attendees: [],
+      wbsName: 'bruh',
+      wbsNum: { carNumber: 1, workPackageNumber: 1, projectNumber: 1 }
+    }
+  ]);
+  console.log(EventDict);
+  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleDropdownClose = () => {
+    setAnchorEl(null);
+  };
+
+  const daysInMonth = (month: Date) => {
+    return new Date(month.getFullYear(), month.getMonth() + 1, 0).getDate();
+  };
+
+  const paddingDays = (month: Date) => {
+    return new Date(month.getFullYear(), month.getMonth(), 0).getDay();
+  };
+
+  const paddingArrayStart = [...Array<number>(paddingDays(displayMonth)).keys()].map(
+    (day) => daysInMonth(new Date(displayMonth.getDate(), displayMonth.getMonth() - 1, displayMonth.getFullYear())) - day
+  );
+  const paddingArrayEnd = [...Array<number>(7 - ((daysInMonth(displayMonth) + paddingDays(displayMonth)) % 7)).keys()].map(
+    (day) => day + 1
+  );
+  const daysThisMonth = paddingArrayStart
+    .concat([...Array(daysInMonth(displayMonth)).keys()])
+    .map((day) => day + 1)
+    .concat(paddingArrayEnd.length < 7 ? paddingArrayEnd : []);
+
+  const unconfirmedDRSDropdown = (
+    <>
+      <NERButton
+        endIcon={<ArrowDropDownIcon style={{ fontSize: 28 }} />}
+        variant="contained"
+        id="unconfirmed-drs-dropdown"
+        onClick={handleClick}
+      >
+        My Unconfirmed DRS
+      </NERButton>
+      <Menu open={!!anchorEl} anchorEl={anchorEl} onClose={handleDropdownClose}>
+        <MenuItem
+          onClick={() => {
+            return;
+          }}
+        >
+          <ListItemIcon>
+            <NoteAddIcon fontSize="small" />
+          </ListItemIcon>
+          Mock Data
+        </MenuItem>
+      </Menu>
+    </>
+  );
+
+  return (
+    <PageLayout title="Design Review Calendar" headerRight={unconfirmedDRSDropdown}>
+      <MonthSelector
+        displayMonth={displayMonth}
+        setDisplayMonth={(date) => {
+          setDisplayMonth(date);
+        }}
+      ></MonthSelector>
+      <Grid container alignItems="center">
+        {['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'].map((day) => (
```

Comment:
I'd move this array to a utils file and call it here. You could also probably make this an enum (there should be a built in method where you can map across all the values in an enum), or dayjs (a package we're gonna add for datetime stuff) might have some built in methods, so we can add that in this PR if it'd be helpful for your stuff)